x-pack/plugins/cases/server/services/cases/index.ts

```diff
@@ -204,35 +207,32 @@ export class CasesService {
    * Returns a map of all cases.
    */
   public async findCasesGroupedByID({
-    unsecuredSavedObjectsClient,
     caseOptions,
   }: {
-    unsecuredSavedObjectsClient: SavedObjectsClientContract;
     caseOptions: FindCaseOptions;
   }): Promise<CasesMapWithPageInfo> {
-    const cases = await this.findCases({
-      unsecuredSavedObjectsClient,
-      options: caseOptions,
-    });
+    const cases = await this.findCases(caseOptions);
 
     const casesMap = cases.saved_objects.reduce((accMap, caseInfo) => {
       accMap.set(caseInfo.id, caseInfo);
       return accMap;
     }, new Map<string, SavedObjectsFindResult<CaseAttributes>>());
 
-    const totalCommentsForCases = await this.getCaseCommentStats({
-      unsecuredSavedObjectsClient,
-      ids: Array.from(casesMap.keys()),
+    const commentTotals = await this.attachmentService.getCaseCommentStats({
+      unsecuredSavedObjectsClient: this.unsecuredSavedObjectsClient,
+      caseIds: Array.from(casesMap.keys()),
     });
 
     const casesWithComments = new Map<string, CaseResponse>();
     for (const [id, caseInfo] of casesMap.entries()) {
+      const { alerts, nonAlerts } = commentTotals.get(id) ?? { alerts: 0, nonAlerts: 0 };
```

Comment:
nit: I think is better to rename `nonAlerts` to something like `userComments` because we have other types of comments (attachments) and you may think that `nonAlerts` contains these other types.

Reply:
Forget my comment. I just review the attachment service code. Indeed, what is returned is all comments except the alerts. I am wondering, does the user assumes the same, or does she assume that this is the count of the total user comments?

Reply:
>  I am wondering, does the user assumes the same, or does she assume that this is the count of the total user comments?

Yeah good question. I could go either way, but `comments` does make me thing of a discussion form style comments `CommentType.user`. So maybe we should only include those?

@peteharverson @academo @monina-n what do you think of when you see the stats under the column `comments` in the cases list page?

Would you expect that to include host isolation attachments?

![image](https://user-images.githubusercontent.com/56361221/152395726-c2f81092-4772-4c61-8028-06352b286782.png)